### PR TITLE
ISPN-12603 link to image getting started

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-12603 

Needs backport to 12.0.x and 11.0.x

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]
LGTM!

File: documentation/src/main/asciidoc/stories/assembly_server_getting_started.adoc [hunk 1]
```diff
@@ -3,6 +3,12 @@
 = Getting Started with {brandname} Server
 Quickly set up {brandname} Server and learn the basics.
 
+//Community
+ifdef::community[]
+image:https://raw.githubusercontent.com/infinispan/infinispan.github.io/develop/assets/images/documentation/icon-get-started.png[Get started icon,height=32]
```
Comment (danberindei) [inline documentation/src/main/asciidoc/stories/assembly_server_getting_started.adoc:8]: Shouldn't we use https://infinispan.org/assets/images/documentation/icon-get-started.png ?

Comment (oraNod) [reply documentation/src/main/asciidoc/stories/assembly_server_getting_started.adoc:8]: Thanks @danberindei That's better!

**Comment by tristantarrant** [comment]
Merged, thanks

**Comment by tristantarrant** [comment]
Pushed to 11.0.x and 12.0.x too

**Pull request merged**